src/selectors/app.js

```diff
@@ -338,3 +340,9 @@ export const getTimelineMarginLeft: Selector<number> = createSelector(
     }
   }
 );
+
+export const getL10nState: Selector<L10nState> = state => state.l10n;
+export const isL10nFetching: Selector<boolean> = state =>
```

Comment:
We usually use `get` + the state in the selectors. It would be better to make it `getIsL10nFetching`.